api/v1/swagger.yaml

```diff
@@ -8144,6 +8724,32 @@ definitions:
         description: >
           Timestamp at which the the write permission was modified
 
+  GeneratorWriterResponse:
+    type: object
+    description: >
+      Users having write permission to the generator
+    properties:
+      generatorId:
+        type: string
+        readOnly: true
+        description: >
+          Id of a write-protected generator
+      userId:
+        type: string
+        readOnly: true
+        description: >
+          Id of a user having write permission to the generator
+      createdAt:
+        readOnly: true
+        type: string
+        description: >
+          Timestamp at which the the write permission was created
+      updatedAt:
```

Comment:
This object is not the `Generator` or the `Writer`, it is the association between the objects, which only seem to be created or destroyed.